src/main/resources/application.conf

```diff
@@ -19,4 +19,10 @@ heroku {
   connTimeoutMs = 50000
   apiToken = dummy-token
   administratorEmails = ["api-maintenance@heroku.com"]
+  defaultReportFile = "report.txt"
+}
+
+github {
+  localDirectory = "prototypes"
+  packageLockReportFile = "package-lock-analysis.txt"
```

Comment:
@JoPintoPaul rather than txt should we use tsv extension

Reply:
@oscarduignan Good question. For some reason the *example* of the report generated by the `generateHerokuReport` is specified as a .txt file, but it doesn't need to be necessarily. I'll investigate.